packages/nuxt/src/core/nitro.ts

```diff
@@ -131,6 +131,8 @@ export async function initNitro (nuxt: Nuxt & { _nitro?: Nitro }) {
         .map(dir => ({ dir }))
     ],
     prerender: {
+      failOnError: true,
+      concurrency: 10,
```

Comment:
We can infer this from user CPU count (also using a power of 2 number makes sure less context switches)